.storybook/preview.jsx

```diff
@@ -1,23 +1,21 @@
-const React = require('react');
-const { ChakraProvider } = require('@chakra-ui/react');
-import { getTheme } from '../src/ui/theme';
-import '@nypl/design-system-react-components/dist/styles.css';
+const React = require("react")
```

Comment:
Looks like there's stuff my linter is doing differently ... what settings are y'all using for jsx?